### PR TITLE
Switch to ubuntu 18.04

### PR DESCRIPTION
**Pull request by dynbit**
Changed `pip install --upgrade pip` to `python -m pip install --upgrade pip` as otherwise it fails.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
-FROM blitznote/debase:16.04
+FROM blitznote/debase:18.04
 
 RUN apt-get update && \
-	apt-get install -y --no-install-recommends git openssh-client make python python-pip python-setuptools && \
+	apt-get install -y --no-install-recommends git openssh-client make python python-pip python-setuptools python3 python3-pip python3-setuptools && \
```
Comment (pranas) [inline Dockerfile]: do we need both python 2 and 3? maybe we can install awscli with pip3?

**Comment by dynbit** [comment]
@pranas tried and seems all is good just with python3, although haven't tested yet that builded image anywhere.

**Comment by pranas** [comment]
Awesome, I like abandoning Python2 and making Python3 the one we use from now on.

**Review by pranas — APPROVED** [review]
👍

**Pull request merged**